style.css

```diff
@@ -1,8 +1,27 @@
- /*
-    ==============================================
+/* ==============================================
                 ****** RESET ******
-    ==============================================
-*/
+  ============================================== */
+
+.svg {
+  fill: aliceblue;
+}
+
+.first {
+  background-color: #f72210;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
```

Comment:
- [ ] This can be done with the bootstrap classes `text-decoration-none` and `text-white`